Firmware/ControladorRobo/ControladorRobo.ino

```diff
@@ -46,8 +47,6 @@ void setup()
 
     HBridge::self();
```

Comment:
Can you have a more complete answer ?
I can't see why are you calling the HBridge::self(), you can't delete a singleton if it's what are you trying to do.